docs/source/transformers/OrderTags.rst

```diff
@@ -0,0 +1,13 @@
+.. _OrderTags:
```

Comment:
Out of curiosity, did you use invoke task to create dummy transformer? Or you created files manually.

Reply:
I used invoke task (however I had to install dev deps manually as the way with '-e .[dev]' didn't installed them for me).
I'll add some more example to this doc tomorrow.

Reply:
Ok thanks - I will check how this command works. If there is anything you think I should improve with this script (which was intended to make life a bit easier when adding stuff ;) ) or with contributing docs in general - let me know.